src/webui/www/private/newfeed.html

```diff
@@ -68,7 +68,7 @@
 <body>
     <div style="padding: 10px 10px 0px 10px;">
         <p style="font-weight: bold;">QBT_TR(Feed URL:)QBT_TR[CONTEXT=RSSWidget]</p>
-        <input type="text" id="feedURL" value="" maxlength="100" style="width: 320px;" />
+        <input type="text" id="feedURL" value="" style="width: 320px;" />
```

Comment:
Could you also omit `value=""`? If yes, please apply to other places too.

Reply:
I see no reason to keep it. Should I also remove it from files that weren't created with my original PR (`upload.html`, `newtag.html`, `rename.html`...)?

Reply:
>Should I also remove it from files that weren't created with my original PR 

Yes, also IMO it is preferable to do it in a separate git commit.